iavl/iavl_proof.go

```diff
@@ -2,11 +2,12 @@ package iavl
 
 import (
 	"bytes"
+	"fmt"
 
 	"golang.org/x/crypto/ripemd160"
 
-	. "github.com/tendermint/tmlibs/common"
 	"github.com/tendermint/go-wire"
+	. "github.com/tendermint/tmlibs/common"
```

Comment:
Note: I know you did not write the module, but  `import .` is deprecated in our codebase.
Proper for tmlibs/common is `cmn` or `common`